pkg/rancher-desktop/backend/k8s.ts

```diff
@@ -14,15 +15,7 @@ export type {
 } from './backend';
 export type { ServiceEntry } from './kube/client';
 
-/**
- * VersionEntry describes a version of K3s.
- */
-export interface VersionEntry {
-  /** The version being described. This includes any build-specific data. */
-  version: semver.SemVer;
-  /** A string describing the channels that include this version, if any. */
-  channels?: string[];
-}
+export type { SemanticVersionEntry } from '@pkg/utils/kubeVersions';
```

Comment:
Why do we need to export this type from `k8s`. Why can't we import from `kubeVersion` everywhere?